Px.Abstractions/Interfaces/IDataSource.cs

```diff
@@ -22,6 +22,13 @@ public interface IDataSource
         /// <returns></returns>
         IPXModelBuilder? CreateBuilder(string id, string language);
 
+        /// <summary>
+        /// Check if table exists
+        /// </summary>
+        /// <param name="tableId"></param>
+        /// <returns></returns>
+        bool TableExists(string tableId, string language, out bool selectionExists);
```

Comment:
Remove the out parameter. We only want:
bool TableExists(string tableId, string language)